.github/workflows/auto_approve_prs.yml

```diff
@@ -0,0 +1,94 @@
+# Auto Approve PRs Workflow
+#
+# This workflow automatically approves pull requests that have specific auto-merge labels attached.
+# It is intended to eventually replace the existing auto_merge workflow by splitting approval and merge
+# into separate workflows.
+#
+# Labels that trigger auto-approval:
+# - "auto-merge": Standard auto-merge label for PRs that should be automatically approved and merged
+# - "auto-merge/bypass-ci-checks": Auto-merge label that bypasses certain CI checks
+#
+# The workflow runs every 15 minutes and searches for open PRs targeting the master branch with
+# these labels, then approves each matching PR using the octavia-bot GitHub App.
+
+name: Auto Approve PRs
+
+on:
+  schedule:
+    # Every 15 minutes
+    - cron: "*/15 * * * *"
+  workflow_dispatch:
+
+concurrency:
+  group: auto-approve-prs
+  cancel-in-progress: false
+
+jobs:
+  find_prs:
+    name: Find PRs to approve
+    runs-on: ubuntu-latest
+    steps:
+      - name: Find PRs with auto-merge labels
+        id: find-prs
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # Search for open PRs with auto-merge labels targeting master branch
+          prs=$(gh api search/issues \
+            --method GET \
+            --field q="repo:airbytehq/airbyte is:pr label:auto-merge OR label:auto-merge/bypass-ci-checks base:master state:open" \
+            --jq '.items[] | {number: .number, title: .title}')
+
+          # Convert to JSON array for matrix
+          if [ -n "$prs" ]; then
+            echo "prs<<EOF" >> $GITHUB_OUTPUT
+            echo "$prs" >> $GITHUB_OUTPUT
+            echo "EOF" >> $GITHUB_OUTPUT
+          else
+            echo "prs=" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Set matrix output
```

Comment:
This should be echoing to terminal so reach step can be debugged if needed. You can use 'tee' or echo.